src/main/java/ee/carlrobert/codegpt/completions/llama/LlamaModel.java

```diff
@@ -82,7 +82,21 @@ public enum LlamaModel {
           HuggingFaceModel.WIZARD_CODER_PYTHON_13B_Q5,
           HuggingFaceModel.WIZARD_CODER_PYTHON_34B_Q3,
           HuggingFaceModel.WIZARD_CODER_PYTHON_34B_Q4,
-          HuggingFaceModel.WIZARD_CODER_PYTHON_34B_Q5));
+          HuggingFaceModel.WIZARD_CODER_PYTHON_34B_Q5)),
+  DBRX(
+      "DBRX",
+      "DBRX is a Mixture-of-Experts (MoE) model with 132B total parameters and 36B live parameters."
+          + "Generation speed is significantly faster than LLaMA2-70B, while at the same time "
+          + "beating other open source models, such as, LLaMA2-70B, Mixtral, and Grok-1 on "
+          + "language understanding, programming, math, and logic.",
+      PromptTemplate.LLAMA,
```

Comment:
I think it uses ChatML prompt template - `PromptTemplate.CHAT_ML`

Reply:
Thx, done